galley/tools/mcps/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	galley "istio.io/istio/galley/pkg/server"
+)
+
+var (
+	serverAddr     = flag.String("server", "127.0.0.1:9901", "The server address")
+	kubeConfig     = flag.String("kubeConfig", "", "The path to kube configuration file")
+	configPath     = flag.String("configPath", "../../../tests/testdata/config", "The config files which galley will monitor")
+	meshConfigFile = flag.String("meshConfigFile", "../../../istioctl/cmd/istioctl/testdata/mesh-config.yaml", "The path for mesh config")
+)
+
+func main() {
```

Comment:
Why can't we use Galley itself for this purpose?

Reply:
To use galley locally, we need modify multiple parameters including probe and validation, etc. This is a minimum sample for running mcp galley server.

Reply:
Sounds like a batch file. :)

Reply:
OK, let me close this PR.